packages/jellyfish-api-core/src/category/poolpair.ts

```diff
@@ -181,3 +218,17 @@ export interface AddPoolLiquidityUTXO {
 export interface PoolShareOptions {
   isMineOnly?: boolean
 }
+
+export interface PoolSwapMetadata {
+  from: string
+  tokenFrom: string
+  amountFrom: number
+  to: string
+  tokenTo: string
+  maxPrice?: number
+}
+
+export interface PoolSwapInputs {
```

Comment:
PoolSwapInputs is a decent name. But for this variable, we write this in other typescript file

```
export interface UTXO {
  txid: string
  vout: number
}
```